server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/Fixture.scala

```diff
@@ -91,4 +108,27 @@ object Fixture {
       |}""".stripMargin
 
   val ACCEPT_RFC8621_VERSION_HEADER: String = "application/json; jmapVersion=rfc-8621"
+
+  val USER_TOKEN: String =
```

Comment:
please provide a way to regenerate these tokens if needed

Reply:
i will try

Reply:
Well, as far as i know in James we only have public key - which used to verify jwt token.So i cannot generate directly jwt as i want.I have a solution for work around but i suggest make it in separate PR.
If im wrong please correct me, if not could you agree with me?

Reply:
Ok, I mean you can just put a comment to explain how you generated these tokens, no need to automate anything

Reply:
ok